scripts/common

```diff
@@ -159,19 +159,36 @@ EOF
 # Takes a variable number of additional repositories as argument.
 # See http://www.scala-sbt.org/0.13/docs/Proxy-Repositories.html
 function generateRepositoriesConfig() {
-  jcenterCacheUrl=${jcenterCacheUrl-"https://scala-ci.typesafe.com/artifactory/jcenter/"}
   sbtRepositoryConfig="$scriptsDir/sbt-repositories-config"
   echo > "$sbtRepositoryConfig" '[repositories]'
   if [[ $# -gt 0 ]]; then
     for i in $(seq 1 $#); do
       echo >> "$sbtRepositoryConfig" "  script-repo-$i: ${!i}"
     done
   fi
+
+  if [ "${TRAVIS}" != "true" ]; then
```

Comment:
I squirreled [one more invocation](https://github.com/scala/scala/pull/6390/files#diff-f1be5674025c26b5d092359fc6c6d9c4R116) away in `mkFreshIvy`, the repo config is still needed. It would be nice to get rid of it by adding the needed resolvers, but I'm not sure where :-/

Reply:
Maybe something like:
```
addSbtPlugin("com.jsuereth" % "sbt-pgp" % "1.1.0")

resolvers ++= Seq(Resolver.typesafeIvyRepo("releases"), Resolver.sbtPluginRepo("releases"))
```

Reply:
We also need that resolver to get STARR. How do I do that?

Reply:
I'm going to punt on this. Would be really nice if we could avoid creating a repositories config, but I don't know how

Reply:
Thanks to some help from @szeiger and @dwijnand, I resolved this too :-). Now I'm rewarded with removing this redundant check.